tests/test_map.cpp

```diff
@@ -251,8 +250,11 @@ TEST_CASE("frozen::map <> std::map", "[map]") {
   SECTION("printing minimal performance requirements") {
     auto std_start = std::chrono::steady_clock::now();
     for (int i = 0; i < 10000; ++i)
-      for (int j = 0; j < 10000; ++j)
-        volatile const auto c = std_map.count(i + j);
+      for (int j = 0; j < 10000; ++j) {
+        benchmark::DoNotOptimize(i);
```

Comment:
why marking the argument as volatile too? Well, I'll watch the video first.

Reply:
Partly, I'm trying to discourage the compiler from unrolling the loop. I want it to believe that after `DoNotOptimize` is called, some crazy assembly could have modified the value of `i` and `j`.

I'm not sure if it was actually necessary in this case or that this ultimately changed the benchmark. My habit now is to usually call `DoNotOptimize` on the inputs and outputs of the code under test, so that it does not get discarded and so that the compiler does not get to take advantage of any patterns it notices in the input, or unroll the loop, since it's not supposed to be able to determine with certainty how many times the loop will happen if I call `DoNotOptimize(j)`, if I understand correctly.